src/librustc/ich/impls_hir.rs

```diff
@@ -203,7 +203,8 @@ impl_stable_hash_for!(struct hir::TyParam {
     default,
     span,
     pure_wrt_drop,
-    synthetic
+    synthetic,
+    attrs
```

Comment:
Please add trailing comma

Reply:
This is inside macro which requires no trailing comma. Otherwise it fails to compile with:
```
error: no rules expected the token `}`
   --> librustc\ich\impls_hir.rs:208:1
    |
208 | });
    | ^

error: aborting due to previous error

error: Could not compile `rustc`.
```
You can look around this place in code - other calls to this macro don't have trailing commas.

Reply:
Didnt notice it was in a macro.